src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -206,8 +227,16 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
         subset_dir (str):
             The name of the directory containing the subsetted pixel data.
             Should be placed in `pixel_output_dir`.
-        norm_vals_name (str):
-            The name of the file to store the 99.9% normalization values
+        norm_vals_name_pre_rownorm (str):
+            The name of the file to store the pre pixel-thresholded norm values
+        norm_vals_name_post_rownorm (str):
+            The name of the file to store the post pixel-thresholded norm values
+        pixel_thresh_name (str):
+            The name of the file to store the pixel threshold value
+        channel_percentile_pre_rownorm (float):
+            Percentile used to normalize channels before pixel thresholding
+        channel_percentile_post_rownorm (float):
+            Percentile used to normalize channels after pixel thresholding
```

Comment:
Same here. For clarify, can we change this to "Percentile used to normalize channels after pixel normalization"